data/upstream.yaml

```diff
@@ -187,6 +199,10 @@ jabber-roster:
   repo: https://github.com/kparal/jabber-roster
   note: |
     Porting will start if deps are available.
+konversation:
+    status: in-progress
+    links:
+        homepage: https://konversation.kde.org/
```

Comment:
I found this on the homepage:

> Konversation now installs an experimental Python scripting support package named 'konversation' into a subdirectory of its application data directory and appends all 'konversation/scripting_support/python' directories found in any KDE application data resource directories (i.e. within $KDEHOME, $KDEDIRS, etc.) to the PYTHONPATH environment variable available in the script execution context, thereby allowing Python scripts executed by Konversation to import the package. The package currently sports modules providing APIs for i18n support and D-Bus communication with Konversation.

> The 'media' script now requires Python 2.6 or higher and is compatible with Python 3.

The commit Dominika found relates to the 2nd paragraph—the 'media' script—therefore I think the package is only partially ported. Should I leave it as 'in-progress' or not?